evalml/pipelines/utils.py

```diff
@@ -644,11 +652,15 @@ def _make_new_component_name(name, component_name, idx=None, pipeline_name=None)
     parameters = copy.deepcopy(parameters) if parameters else {}
     final_components = []
     used_names = []
-    component_graph = (
-        {"Label Encoder": ["Label Encoder", "X", "y"]}
-        if is_classification(problem_type)
-        else {}
+
+    prior_components = {} if not prior_components else prior_components
```

Comment:
most definitely but I'm trying to avoid having an empty dictionary as a parameter. But since we don't return `prior_components` it should be safe to do so. Will update!